src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -515,8 +515,14 @@ public async Task<IActionResult> OptInStandardVersion(string referenceNumber, st
 
         [HttpPost("standard/opt-in", Name = OptInStandardVersionRoutePost)]
         [PrivilegeAuthorize(Privileges.ApplyForStandard)]
+        [ModelStatePersist(ModelStatePersist.Store)]
         public async Task<IActionResult> OptInStandardVersion(OptInStandardVersionViewModel model)
         {
+            if (!ViewData.ModelState.IsValid)
```

Comment:
!ModelState.IsValid - the ViewData prefix is not required and inconsistent with other places